BLSQ/iaso
# IA-3471: adding oucrc permissions selection

Org unit change request config permission is not visible in the dashboard while editing an user

Related JIRA tickets : IA-3471
## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes

- adapt menu permission to display entry in the menu (currently set to source version)
- added `iaso_org_unit_change_request_configurations` to `PERMISSIONS_PRESENTATION`

## How to test

Edit a profile and check that you can assign iaso_org_unit_change_request_configurations to a user

## Print screen / video

<img width="1094" alt="Screenshot 2024-10-01 at 15 18 28" src="https://github.com/user-attachments/assets/71f62c79-8593-4372-a8a8-50c17304212b">
